kotlin-sqlobject/src/test/kotlin/org/jdbi/v3/kotlin/KotlinSqlObjectPluginTest.kt

```diff
@@ -57,6 +58,10 @@ class KotlinSqlObjectPluginTest {
         @SqlQuery("select id, name from something where id=:id")
         fun findById(id: Int): Thing
 
+        @SqlQuery("select id as t_id, name as t_name from something where id=:id")
+        @RegisterKotlinMapper(Thing::class, "t_")
```

Comment:
Fun fact: because Jdbi elides underscores when comparing column names, this would also work with just `prefix = "t"`